packages/ibm-products/src/components/InlineTip/InlineTip.test.js

```diff
@@ -0,0 +1,192 @@
+  it('renders a component InlineTip', () => {
+    render(<InlineTip title={title}>{children}</InlineTip>);
+    expect(screen.getByRole('main')).toHaveClass(blockClass);
+  });
+
+  it.skip('has no accessibility violations', async () => {
+    const { container } = render(
+      <InlineTip title={title}>{children}</InlineTip>
+    );
+    await expect(container).toBeAccessible(componentName);
+    await expect(container).toHaveNoAxeViolations();
+  });
+
+  it(`renders children`, () => {
+    render(<InlineTip title={title}>{children}</InlineTip>);
+    screen.getByText(children);
+  });
+
+  it(`renders ghost button`, () => {
+    render(
+      <InlineTip
+        title={title}
+        contentCTA={
+          <InlineTipButton onClick={() => {}}>{buttonLabel}</InlineTipButton>
+        }
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(buttonLabel);
+  });
+
+  it(`renders link`, () => {
+    render(
+      <InlineTip
+        title={title}
+        contentCTA={
+          <InlineTipLink href="https://www.ibm.com" target="_blank">
+            {linkLabel}
+          </InlineTipLink>
+        }
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(linkLabel);
+  });
+
+  it(`renders icon button with 'x' icon`, () => {
+    render(
+      <InlineTip title={title} onClose={() => {}}>
+        {children}
+      </InlineTip>
+    );
+    expect(
+      document.querySelector('.c4p--inline-tip__close-icon')
+    ).toBeInTheDocument();
+  });
+
+  it(`renders tertiary button with 'crossroads' icon`, () => {
+    render(
+      <InlineTip title={title} tertiaryButtonLabel={tertiaryButtonLabel}>
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(tertiaryButtonLabel);
+  });
+
+  it(`renders collapsible, collapsed`, () => {
+    render(
+      <InlineTip
+        title={title}
+        collapsible
+        expandButtonLabel={readMoreLabel}
+        tertiaryButtonLabel={readLessLabel}
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(readMoreLabel);
+  });
+
+  it(`renders collapsible, expanded`, () => {
+    render(
+      <InlineTip
+        title={title}
+        collapsible
+        expandButtonLabel={readMoreLabel}
+        tertiaryButtonLabel={readLessLabel}
+      >
+        {children}
+      </InlineTip>
+    );
+    userEvent.click(document.querySelector('button'));
+    screen.getByText(readLessLabel);
+  });
+
+  it(`renders an image`, () => {
+    render(
+      <InlineTip
+        title={title}
+        media={{ render: () => <img alt="img" src={InlineTipImage} /> }}
+      >
+        {children}
+      </InlineTip>
+    );
+    expect(document.querySelector('img')).toBeInTheDocument();
+  });
+
+  it(`renders in the narrow format`, () => {
+    render(
+      <InlineTip title={title} narrow>
+        {children}
+      </InlineTip>
+    );
+    expect(
+      document.querySelector('.c4p--inline-tip__narrow')
```

Comment:
```suggestion
    expect(screen.getByRole('complementary')).toHaveClass(blockClass);
```
I think we often do something like this? (Assuming we use `complementary` — current state would use `main`, of course.)